packages/ember-data/tests/integration/serializers/embedded_records_mixin_test.js

```diff
@@ -1320,3 +1322,76 @@ test("serializing relationships with an embedded and without calls super when no
   ok(calledSerializeBelongsTo);
   ok(calledSerializeHasMany);
 });
+
+test("embedded records should be created in multiple stores", function() {
```

Comment:
Can you extract this into a specific module that tests support for multiple stores. I don't think this is embedded records specific